Resync web-platform-tests/websockets from upstream of 7ebd92cb6591376b5fe7cb32326a03539c4e60c2 (2024-04-17)

fujii opened the pull request:
#### 3bf4325c36aeab5781e60737c685a40edfc46c15
<pre>
Resync web-platform-tests/websockets from upstream of 7ebd92cb6591376b5fe7cb32326a03539c4e60c2 (2024-04-17)
<a href="https://bugs.webkit.org/show_bug.cgi?id=280731">https://bugs.webkit.org/show_bug.cgi?id=280731</a>

Reviewed by Anne van Kesteren.

Upstream commit: <a href="https://github.com/web-platform-tests/wpt/commit/7ebd92cb6591376b5fe7cb32326a03539c4e60c2">https://github.com/web-platform-tests/wpt/commit/7ebd92cb6591376b5fe7cb32326a03539c4e60c2</a>

This was the previous commit of wpt&apos;s pywebsocket3 migration.
<a href="https://github.com/web-platform-tests/wpt/commit/a7a594d8c03a850fd99bab1fd85bb6fe310081a2">https://github.com/web-platform-tests/wpt/commit/a7a594d8c03a850fd99bab1fd85bb6fe310081a2</a>

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/web-platform-tests/websockets/...: Resynced.
* LayoutTests/tests-options.json:

Canonical link: <a href="https://commits.webkit.org/284820@main">https://commits.webkit.org/284820@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/306e98e7c42ac5aa6e9893a165b537d4516d1ee8

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/70523 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/49929 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/23288 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/74612 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/21701 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/57729 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/21541 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/55878 "Passed tests") | [✅ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/14345 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/73589 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/45407 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/60796 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/36339 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/42066 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/64/builds/18231 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/20062 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/63999 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/63/builds/18586 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/76331 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/87/builds/14749 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/62/builds/17811 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/63612 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/86/builds/14792 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/60863 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/63547 "Passed tests") | 
| | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/88/builds/11578 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/119/builds/5220 "Passed tests") | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/10818 "Built successfully and passed tests") | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/45731 "Built successfully") | [✅ 🛠 mac-safer-cpp](https://ews-build.webkit.org/#/builders/120/builds/498 "Built successfully") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/46805 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/48082 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/46547 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

annevk reviewed (approved):
Why not synchronize it all at once to the current version? PR gets too big?

Also, are you using `--clean-dest-dir`? There might well be leftover files to clean up.

fujii committed:
Resync web-platform-tests/websockets from upstream of 7ebd92cb6591376b5fe7cb32326a03539c4e60c2 (2024-04-17)
https://bugs.webkit.org/show_bug.cgi?id=280731

Reviewed by Anne van Kesteren.

Upstream commit: https://github.com/web-platform-tests/wpt/commit/7ebd92cb6591376b5fe7cb32326a03539c4e60c2

This was the previous commit of wpt's pywebsocket3 migration.
https://github.com/web-platform-tests/wpt/commit/a7a594d8c03a850fd99bab1fd85bb6fe310081a2

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/web-platform-tests/websockets/...: Resynced.
* LayoutTests/tests-options.json:

Canonical link: https://commits.webkit.org/284820@main
